docs/DEVELOPMENT.md

```diff
@@ -189,7 +189,11 @@ is because we do not have health checking set up yet. Please refer <https://gith
 
 ### Using `curl` + `kubectl port-forward`
 
-See the available REST endpoints in the [OpenAPI specification](api/rest-api-spec.md) docs. The API request URL is of the format `https://{server_url}/apis/results.tekton.dev/v1alpha2/parents/{name/path-to-the-resource}`. For debugging server_url is `localhost:port-exposed`.
+See the available REST endpoints in
+the [OpenAPI specification](https://petstore.swagger.io/?url=https://raw.githubusercontent.com/tektoncd/results/main/docs/api/openapi.yaml).
```

Comment:
An interesting thought - running our own Swagger instance within https://tekton.dev/docs/

Reply:
Yes, I was thinking of proposing this in our next WG call. Let me add it.